data/weapons.txt

```diff
@@ -652,15 +652,15 @@ outfit "Meteor Missile Launcher"
 		"hit effect" "missile hit"
 		"inaccuracy" 5
 		"velocity" 11
-		"lifetime" 490
+		"lifetime" 240
 		"reload" 80
 		"firing energy" 1
 		"firing heat" 20
 		"acceleration" 1
 		"drag" .1
-		"turn" 1.35
-		"homing" 3
-		"infrared tracking" .8
+		"turn" 1.8
+		"homing" 1
```

Comment:
> but it seems that this would also make it very unforgiving to use meteors and miss.

That was kind of the goal. Also, now infrared missiles become more likely to hit as the distance to the target decreases. I feel like it works out pretty well in practice.